src/docs/get-started/codelab-web.md

```diff
@@ -765,6 +766,10 @@ class _AnimatedProgressIndicatorState extends State<AnimatedProgressIndicator>
   }
 }
 ```
+
+The [didUpdateWidget][] function is called to 
+update our `AnimatedProgressIndicatorState` whenever
+our `AnimatedProgressIndicator` changes.
```

Comment:
Delete both instances of "our".

Reply:
done, also caught a separate usage of "our"